environment.yml

```diff
@@ -0,0 +1,34 @@
+# To use:
+#   $ conda env create -f environment.yml  # `mamba` works too for this command
+#   $ conda activate scipy-dev
+name: scipy-dev
+channels:
+  - conda-forge
+dependencies:
```

Comment:
NumPy is missing no? Or I am missing something?

Reply:
It got pulled in by Matplotlib, but yeah .... oops. Fixed now.